stages/org.osbuild.systemd.unit.create

```diff
@@ -44,6 +48,17 @@ SCHEMA = r"""
         "type": "string",
         "pattern": "^[\\w:.\\\\-]+[@]{0,1}[\\w:.\\\\-]*\\.service$"
     },
+    "unit-type": {
+        "type": "string",
+        "enum": ["system", "global"],
+        "default": "system",
+        "description": "Selects between systemd system or global unit to add dropin"
+    },
+    "persistent": {
+        "type": "boolean",
+        "default": false,
+        "description": "Define if systemd unit file should survive rpm-ostree upgrade"
+    },
```

Comment:
The option name doesn't make much sense here without the description about rpm-ostree.  Since the stage isn't ostree-specific, I would change the name to something that implies _what_ happens (instead of _why_).
In the non-ostree case, the option doesn't even affect the persistence of the unit, only the location, so it's even misleading in that case.

I'm not sure what a good name for it would be though.  Perhaps simply `etc` would be enough (boolean, default false) or maybe follow what we do with `unit-type`:
```json
"location": {
    "type": "string",
    "enum": ["usr", "etc"],
    "default": "usr"
}
```

@mvo5 @supakeen ping for thoughts.

Reply:
Location as a key one I like, for values the other option would be `vendor` (`usr`) vs `user` (`etc`) if we want to follow what should go where though I prefer the `usr`/`etc` values.

However, `location` is also kinda funky. If I take a look at the `systemd.unit` manpages they call this the unit load path. So perhaps `unit-save-path`?

Reply:
I will modify `bool` to `string` as it makes more sense considering the diverse ecosystem.
we can choose any  key from: `unit-path`  , `unit-load-path`  , `unit-save-path` they go well with another key `unit-type` that is used in multiple schema.

Reply:
updated key as `unit-path`.